RealEstate/apps/core/templates/core/categories.html

```diff
@@ -85,11 +45,11 @@
 	{% endfor %}
 </ul>
 <center>
-    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#addCategory" style="margin:30px">Add Category</button>
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#addCategory" id="button-margin">Add Category</button>
 </center>
 
 <!-- Add Category Modal -->
-<div id="addCategory" class="modal fade" role="dialog" style="height:auto">
+<div id="addCategory" class="modal fade" role="dialog" id="modalHeight">
```

Comment:
Two id's here. That's probably the reason there's some weird behavior. modalHeight feels like a class anyway. Let's just stick it in with the other two classes.